### DISCUSSION TITLE
Any other way to get infusion/ingredient drugs in the ICU other than items_ID?

### DISCUSSION THREAD

**Discussion by ededededb** [MIT-LCP/mimic-code#1575] [MIMIC-IV] [answered]
I'm a bit confused because the drug I am trying to find is quite common (tranexamic acid), and I am not able to find it on d_items. Does this mean that I won't be able to find which patients got tranexamic acid on the inputevents file? 

Also, one more question - how to find mortality (in and out of the ICU - for up to one year after ICU admission)?

**Comment by alistairewj** [comment, marked as answer]
One thing that's important to note is that *inputevents* only contains continuous infusions from a fixed list determined apriori. This means that there are a few medications which aren't in *inputevents*, and you'd have to look elsewhere (*prescriptions* for intention to treat, *emar* for administrations).

That being said, I don't think they use TXA. They use aminocaproic acid (Amicar).

```sql
select itemid, label n from `physionet-data.mimiciv_icu.d_items`
where lower(label) like '%tranexamic%'
or lower(label) like '%lysteda%'
or lower(label) like '%txa%'
or lower(label) like '%aminocaproic%'
or lower(label) like '%amicar%'
```

itemid | label
-- | --
229058 | Aminocaproic acid (Amicar)
229249 | Aminocaproic acid (Amicar)

But it's not that common:

```sql
select itemid, count(*) AS n from `physionet-data.mimiciv_icu.inputevents`
where itemid IN
(
  229058,
  229249
)
group by 1
```

Row | itemid | n |  
-- | -- | -- | --
1 | 229058 | 265

Mortality is in the *patients* table (`dod`).

P.S. One discussion post per question is better, since we can only accept one answer per question, and it makes search easier.
